components/raftstore-v2/tests/integrations/test_conf_change.rs

```diff
@@ -47,6 +55,50 @@ fn test_simple_change() {
         req.get_header().get_peer().get_id()
     );
 
+    let mut raft_msg = RaftMessage::default();
```

Comment:
Why need this? I think a message append should be sent immediately. Even if it's not sent, you can trigger the message by ticking a heartbeat.

Reply:
yes, I have use raft tick to trigger the snapshot